### PR TITLE
Skip `default` check for outputs

### PR DESCRIPTION
**Pull request by viralpraxis**
Resolves https://github.com/sunny/actor/issues/131

### CODE REVIEW

File: spec/actor_spec.rb [hunk 1]
```diff
@@ -721,6 +721,15 @@
       end
     end
 
+    context "with unset output" do
+      specify do
```
Comment (sunny) [inline spec/actor_spec.rb:725]:
```suggestion
    context "with unset output and allow_nil: true" do
      it "succeeds" do
```

File: lib/service_actor/checkable.rb [hunk 2]
```diff
@@ -6,6 +6,15 @@ def self.included(base)
...
 
   module PrependedMethods
+    CHECK_CLASSES = [
+      ServiceActor::Checks::TypeCheck,
+      ServiceActor::Checks::MustCheck,
+      ServiceActor::Checks::InclusionCheck,
+      ServiceActor::Checks::NilCheck,
+      ServiceActor::Checks::DefaultCheck
+    ].freeze
+    private_constant :CHECK_CLASSES
```
Comment (sunny) [inline lib/service_actor/checkable.rb:16]: We should probably remove the `check_classes` method then.

**Review by sunny — CHANGES_REQUESTED** [review]
Can you also add an entry to the CHANGELOG? 🙏🏻

File: lib/service_actor/checkable.rb (hunk 2 above)
Comment (viralpraxis) [reply lib/service_actor/checkable.rb:16]: Absolutely, I just forgot

**Review by sunny — APPROVED** [review]
Perfect! 😍 Thank you for squashing that bug 👏🏻

**Pull request merged**